unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Definitions/Material.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+import "common/id.proto";
+option (ecs_component_id) = 1017;
```

Comment:
This can be on `https://github.com/decentraland/protocol`

To test from a PR from `https://github.com/decentraland/protocol`, go to `ProtobufEditor.cs` and look for this line:

```
private const string FIXED_NPM_PACKAGE_LINK = "";
```

you must apply the npm package URL (which the bot from @dcl/protocol provides you when you open a PR)

Reply:
yeap, I will be adding it to the protocol repo